### PR TITLE
Tziminadis/card revamping

### PR DESCRIPTION
**Pull request by Tziminadis**

### CODE REVIEW

**Comment by Tziminadis** [comment]
I tried out an ideia in the lower button of the card and would like to know what you think about it. Couldn't get it to follow the "body__card" element radius, though.

**Comment by felipprodrigues** [comment]
Overall your PR looks good but there were some major changes on the design and would be pleased if you could follow it. Not sure this is the final one but I think "design wise" we are getting there  

Screen Shot 2021-04-18 at 8 30 31 PM

File: _src/css/_body.scss [hunk 1]
```diff
@@ -7,60 +7,106 @@
...
   &__frame {
+    width: 30%;
     display: flex;
     flex-direction: column;
     position: relative;
+    justify-content: center;
+    align-items: center;
   }
 
   &__frame::after{
```
Comment (felipprodrigues) [inline _src/css/_body.scss]: We no longer will be using this pseudo class

**Review by felipprodrigues — CHANGES_REQUESTED** [review]

File: _src/css/_body.scss [hunk 2]
```diff
@@ -17,96 +17,50 @@
...
-  }
-
   &__frame {
     width: 30%;
     display: flex;
     flex-direction: column;
     position: relative;
     justify-content: center;
     align-items: center;
+    bottom: 15px;
```
Comment (felipprodrigues) [inline _src/css/_body.scss]: This rule shouldn't be here. It is pushing the entire component out of its parent

File: _src/css/_body.scss [hunk 3]
```diff
@@ -17,96 +17,50 @@
...
 
   &__title{
+    font-size: 25px;
     display: flex;
-    justify-content: center;
+    justify-content: flex-start;
   }
 
   &__text{
+    margin-top: 35px;
```
Comment (felipprodrigues) [inline _src/css/_body.scss]: There's no need for the margin to be this much. We can use the default marging of this element to set the correct spacing between the title and text.

File: _src/body/_body.jade [hunk 4]
```diff
@@ -1,13 +1,10 @@
...
-        h3.body__title DOGOLA
-        p.body__text Beyond the anthills of the dawning of this plague, said I've lost my way. Even if this cul de sac did pay. To reach inside a vault whatever be the cost. Sterling clear. Blackend ice. When they drag the lake theres nothing left at all 
-    .body__collapse
-      span.body__icon
+    .body__frame
+      img.body__image(src="assets/dogola.png", alt="OOPSS")
+    .body__content
+      h3.body__title DOGOLA
+      p.body__text Beyond the anthills of the dawning of this plague, said I've lost my way. Even if this cul de sac did pay. To reach inside a vault whatever be the cost. Sterling clear. Blackend ice. When they drag the lake theres nothing left at all 
   .body__card
```
Comment (felipprodrigues) [inline _src/body/_body.jade:8]: We can size 50px down the body__card element. It should fit better the entire content held in it.

**Review by felipprodrigues — CHANGES_REQUESTED** [review]

**Review by felipprodrigues — APPROVED** [review]

**Pull request merged**